### PR TITLE
Improve Process and DNS API docs style

### PR DESCRIPTION
**Pull request by galpeter**
* Removed the backtick from the property/method name headers.
* Corrected 'default' value style in Process docs.

### CODE REVIEW

File: docs/api/IoT.js-API-DNS.md [hunk 1]
```diff
@@ -16,30 +16,30 @@ implemented on top of the underlying operation system facilities. Please see
 [Implementation considerations section](#implementation-considerations).
 
 The following properties are flags which can be passed as hints for the
-[`dns.lookup()`](#dnslookuphostname-options-callback) method.
+[dns.lookup()](#dnslookuphostname-options-callback) method.
```
Comment (zherczeg) [inline docs/api/IoT.js-API-DNS.md]: We are keeping the aphostrophe in the references, but not in the section headers.

File: docs/api/IoT.js-API-DNS.md [hunk 2]
```diff
@@ -18,28 +18,28 @@ implemented on top of the underlying operation system facilities. Please see
...
-* `options {Object} | {number}`
-  * `family {number}` The record family. If specified must be 4. Specifies that IPv4 addresses should be returned.
-  * `hints {number}` One or more supported getaddrinfo flags. Multiple flags may be passed by bitwise `OR`ing their values.
-* `callback {Function}`
-  * `err {Error}` If there is no error the value is `null`.
-  * `address {string}` A string representation of an IPv4 address.
-  * `family {number}` 4, denoting the family of `address`.
+### dns.lookup(hostname[, options], callback)
+* `hostname` {string} Hostname to be resolved.
+* `options` {Object} | {number}
```
Comment (zherczeg) [inline docs/api/IoT.js-API-DNS.md]: {Object|number}

**Comment by galpeter** [comment]
I've fixed the PR

**Comment by yichoi** [comment]
LGTM

**Pull request merged**